src/core/ApolloClient.ts

```diff
@@ -213,6 +220,7 @@ export class ApolloClient<TCacheShape> implements DataProxy {
       fragmentMatcher,
       name: clientAwarenessName,
       version: clientAwarenessVersion,
+      dataMasking = false,
```

Comment:
Since you `!!` it anyways, you don't need the default value - `undefined` will work nicely and default values transpile to a lot of code :)
```suggestion
      dataMasking,
```